output/analysis/plots/sconscript.log

```diff
@@ -1,4 +1,11 @@
-*** Builder log created: {2021-05-30 10:42:17}
-*** Builder log completed: {2021-05-30 10:42:31}
-[Warning: MATLAB has disabled some advanced graphics rendering features by switching to software OpenGL. For more information, click <a
-href="matlab:opengl('problems')">here</a>.] 
+*** Builder log created: {2022-03-30 13:49:44}
+*** Builder log completed: {2022-03-30 13:50:05}
+[Warning: Function plot has the same name as a MATLAB builtin. We suggest you rename the
```

Comment:
Maybe we should rename the function to avoid this warning going forward?

Reply:
@jmshapir Done in 0d23e83, error no longer occurs.